utils.py

```diff
@@ -64,6 +65,24 @@ def to_snake(name):
     return name.replace("-", "_")
 
 
+def to_nice_yaml(data):
+    """Implement a filter for Jinja 2 templates to render human readable YAML."""
+    return yaml.dump(data, indent=2, allow_unicode=True, default_flow_style=False)
+
+
+def merge_toml(template, plugin_root_dir, relative_path, config):
+    basename, merge_key = relative_path.split(".toml.", maxsplit=1)
+    data = tomlkit.loads(template.render(**config))
+    if merge_key in data:
+        path = pathlib.Path(plugin_root_dir) / f"{basename}.toml"
+        old_toml = tomlkit.load(path.open())
+        if merge_key not in old_toml:
+            old_toml[merge_key] = data[merge_key]
+        else:
+            old_toml[merge_key].update(data[merge_key])
+        tomlkit.dump(old_toml, path.open("w"))
+
+
```

Comment:
Can you add a docstring minimum example of the desired behavior for this?
So we can glimps into this in the future and know what to expect.